yt/frontends/tipsy/io.py

```diff
@@ -170,11 +173,11 @@ def _read_particle_data_file(self, data_file, ptf, selector=None):
             p = np.fromfile(f, self._pdtypes[ptype], count=count)
             auxdata = []
             for afield in afields:
-                aux_fh[afield].seek(aux_fields_offsets[afield][ptype])
+                aux_fh(afield).seek(aux_fields_offsets[afield][ptype])
                 if isinstance(self._aux_pdtypes[afield], np.dtype):
                     auxdata.append(
                         np.fromfile(
-                            aux_fh[afield], self._aux_pdtypes[afield], count=count
+                            aux_fh(afield), self._aux_pdtypes[afield], count=count
                         )
                     )
                 else:
```

Comment:
this else block needs to be updated to use the new `aux_fh` function as well (see test failures)